specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Is a CVR going to _contain_ a `ballot_manifest_info`, or be associated with one in a table row? (recall that a `ballot_manifest_info` is itself a set of something like 6 different data items, so while we could index on that, it might be cumbersome.